modular_skyrat/modules/contractor/code/datums/outfit.dm

```diff
@@ -0,0 +1,3 @@
+// SKYRAT EDIT ADDITION - Underwear and bra split
+/datum/outfit
```

Comment:
No need for Skyrat comments when you're in modular files. Please add documentation to that `bra` variable, though, just like I showed you how to do elsewhere.
```suggestion
/datum/outfit
```